src/routes/contact.js

```diff
@@ -179,6 +179,7 @@ router.post('/send', async (req, res) => {
             name: contact.name,
```

Comment:
If I'm reading the overall flow correctly, the entity IDs and contact IDs are never used in this method. It also appears that if they are provided, the e-mail is not sent.

Am I missing something?

Reply:
You're correct. This was left with a future state in mind that was put on the back burner. I'm going to revisit this now though.